### PR TITLE
NAS-121688 / 23.10 / test_account_ssh_key.py

### PR DESCRIPTION
**Pull request by themylogin**

### CODE REVIEW

**Comment by themylogin** [comment]
time 3:00

**Review by yocalebo — APPROVED** [review]
Looks good, but let's wait on @anodos325 final review before merge, please.

**Comment by anodos325** [comment]
No commit message explaining change.

**Comment by themylogin** [comment]
@anodos325 @yocalebo the ticket opener expects exactly the opposite, what should we do here?

**Review by anodos325 — APPROVED** [review]

**Comment by anodos325** [comment]
I'd go ahead with deletion. I'll look at ticket, but I don't think the ticket opener's request makes sense.

File: src/middlewared/middlewared/plugins/account.py
Comment (yocalebo) [inline src/middlewared/middlewared/plugins/account.py:902]: Blocking IO

File: src/middlewared/middlewared/plugins/account.py
Comment (yocalebo) [inline src/middlewared/middlewared/plugins/account.py:865]: This doesn't need to be an option. When a user is deleted, we should always remove the .ssh dir.

**Review by yocalebo — CHANGES_REQUESTED** [review]

**Review by yocalebo — APPROVED** [review]

**Comment by themylogin** [comment]
time 1:00

**Pull request merged**
